src/Controller/Api/UserCanUnsubscribeController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use AppBundle\Entity\Adherent;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+class UserCanUnsubscribeController extends Controller
+{
+    /**
+     * @Route("/can_unsubscribe/{uuid}", name="api_can_unsubscribe")
```

Comment:
you don't use this route?

Reply:
Auth app use this route to check if the user can unsubscribe.